### PR TITLE
Fix #5142 Edit text foreground and background contrast ratio issue in Light mode 

### LINKED ISSUE #5142: [BUG]: Edit text foreground and background contrast ratio issue in Light mode
### Describe the bug

 Edit text foreground and background contrast ratio issue in admin_pin_activity

### Steps To Reproduce

1. Click on set up Multiple Profiles.
2. Edit text has accessibility issue (foreground and background contrast ratio issue) in light mode.

### Expected Behavior

Accessability scanner should not show accessability issue for Edit text.

### Screenshots/Videos

afterlightmode

### PR DESCRIPTION
**Pull request by Vishwajith-Shettigar**
## Explanation
Fix #5142, changed text input default stroke color.
Added color resource file names component_color_text_layout.xml in order to give stroke color

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

Berore fix
| Light mode| Dark mode|
|--------|--------|
| contarstlight_before|contastdark_before| 

Afterfix
| Light mode| Dark mode|
|--------|--------|
|newnew|contrastdark_after| 

Screen recording after fix

Light mode

https://github.com/oppia/oppia-android/assets/76042077/943629a3-2b4a-4459-bbf9-0f0246860ffd

Dark mode

https://github.com/oppia/oppia-android/assets/76042077/c85256a0-5f99-4577-82e4-b95ce0d24027

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by Vishwajith-Shettigar** [comment]
@adhiamboperes PTAL

**Comment by Vishwajith-Shettigar** [comment]
@adhiamboperes  Could you please review the failed checks and provide guidance on how to resolve them? Thank you.

File: app/src/main/res/color/component_color_textlayout.xml
Comment (adhiamboperes) [inline app/src/main/res/color/component_color_textlayout.xml:1]: Please use proper snake case on the file name

File: app/src/main/res/color/component_color_textlayout.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:color="@color/color_palette_edit_text_inactive_color" android:state_focused="false"/>
+  <item android:color="@color/color_palette_edit_text_active_color" android:state_focused="true"/>
+</selector>
```
Comment (adhiamboperes) [inline app/src/main/res/color/component_color_textlayout.xml]: Add newline to eof

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @Vishwajith-Shettigar! 
I have left some suggestions, but this looks okay.

Could you also just add the screen recordings so that we can see the active/inactive state color changes in action?

File: app/src/main/res/color/component_color_textlayout.xml (hunk 1 above)
Comment (Vishwajith-Shettigar) [reply app/src/main/res/color/component_color_textlayout.xml]: okay

File: app/src/main/res/color/component_color_textlayout.xml
Comment (Vishwajith-Shettigar) [reply app/src/main/res/color/component_color_textlayout.xml:1]: component_color_text_layout.xml is that okay ?

**Comment by Vishwajith-Shettigar** [comment]
> Thanks @Vishwajith-Shettigar! I have left some suggestions, but this looks okay.
> 
> Could you also just add the screen recordings so that we can see the active/inactive state color changes in action?

I added Screen recordings,thank you.

**Review by adhiamboperes — APPROVED** [review]
This LGTM, thanks @Vishwajith-Shettigar!

**Pull request merged**